packages/web/components/JournalyEditor/Toolbar/Toolbar.tsx

```diff
@@ -29,176 +29,197 @@ import ToggleMarkButton from './ToggleMarkButton'
 import ToolbarButton from './ToolbarButton'
 import InsertImageButton from './InsertImageButton'
 import { options, isTableActive } from '../helpers'
+import SwitchToggle from '@/components/SwitchToggle'
+import { useTranslation } from '@/config/i18n'
+import useGetWindowSize from '@/hooks/useGetWindowSize'
+import useIntersectionObserver from '@/hooks/userIntersectionObserver'
 
 type ToolbarProps = {
   allowInlineImages: boolean
+  shouldPlayTypewriterSounds: boolean
+  onToggleShouldPlayTypewriterSounds: () => void
 }
 
-const Toolbar = ({ allowInlineImages }: ToolbarProps) => {
+const Toolbar = ({
+  allowInlineImages,
+  shouldPlayTypewriterSounds,
+  onToggleShouldPlayTypewriterSounds,
+}: ToolbarProps) => {
+  const { t } = useTranslation('j-editor')
+
   const editor = useSlate()
   const isEditorFocused = useFocused()
-  const toolbarRef = useRef<HTMLDivElement>(null)
-  const [isFixed, setIsFixed] = useState(false)
-  // Save the initial distance that the toolbar is from the top of the page
-  const originalElementOffsetTop = toolbarRef?.current?.offsetTop as number
-  // Save height of toolbar so that when it's fixed and removed from document flow,
-  // the parent takes the same height and surrounding content won't collapse
-  const fixedDistanceFromTop = 20
-  const toolbarHeight = toolbarRef.current?.getBoundingClientRect().height || 56
-  const toolbarClasses = classNames('editor-toolbar-container', { 'is-fixed': isFixed })
+  const stickyToolbarOffset = 20
```

Comment:
Having this and "toolbarStickyOffset" is a bit confusing. Maybe just get rid of this variable and make `toolbarStickyOffset = <condition> ? -92 : -20` below?